src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala

```diff
@@ -17,7 +17,7 @@ class FetchArticlesController(fetchSavedArticles: FetchSavedArticles)(implicit e
          syncedPrefs.savedArticles.map( sp =>
```

Comment:
I find foreach assists reading as it's obvious that the return isn't intended to be used